packages/vite-flow/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Callstack
```

Comment:
Callstack? :D

Reply:
didn't notice that, it was just there after copying the vite swc package in the workspace!